src/components/HomePage/Home.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getEmojis } from '../../redux/reduxHome/ApiEmojis';
+import './home.css';
+import Nav from '../navbar/nav';
+import Headershow from './Header';
+import EmojiList from './EmojiList';
+
+const GetInformation = () => {
+  const dispatch = useDispatch();
+  const emojisData = useSelector((store) => store.emojis);
+
+  useEffect(() => {
+    if (!emojisData.length) {
+      dispatch(getEmojis());
+    }
+  });
+
+  const handlefilter = (e) => {
+    dispatch(getEmojis(e.target.value));
+  };
+
+  return (
+    <section className="Container-Home">
+      <Nav name="Emoji App" />
+      <div className="content-select">
+        <select onChange={handlefilter} name="select" className="menu-filter">
```

Comment:
Great job @ErikStoupignan 🤝

- [ ] [Optional] Great job implementing the filter action which you created using a dropdown menu. However, you could have created a search bar and made sure the user can search by category or by name. You could have achieved this using the `filter()` method.